Tessian/Tessian.manifest.json

```diff
@@ -26,12 +26,11 @@
     "requirements": [],
     "requiresInstallationInstructions": false,
     "screenshotURLs": [
-        "https://sumologic-partner-appsdata.s3.amazonaws.com/dashboards/Tessian/tessian_guardian_dashboard.jpg",
-        "https://sumologic-partner-appsdata.s3.amazonaws.com/dashboards/Tessian/tessian_defender_dashboard.jpg",
-        "https://sumologic-partner-appsdata.s3.amazonaws.com/dashboards/Tessian/tessian_enforcer_dashboard.jpg",
-        "https://sumologic-partner-appsdata.s3.amazonaws.com/dashboards/Tessian/tessian_overview_dashboard.jpg",
-        "https://sumologic-partner-appsdata.s3.amazonaws.com/dashboards/Tessian/tessian_constructor_dashboard.jpg"
+        "https://sumologic-partner-appsdata.s3.amazonaws.com/dashboards/Tessian/tessian_overview_dashboard.png",
+        "https://sumologic-partner-appsdata.s3.amazonaws.com/dashboards/Tessian/tessian_guardian_dashboard.png",
+        "https://sumologic-partner-appsdata.s3.amazonaws.com/dashboards/Tessian/tessian_defender_dashboard.png",
+        "https://sumologic-partner-appsdata.s3.amazonaws.com/dashboards/Tessian/tessian_architect_dashboard.png"
     ],
-    "uuid": "9ddbe84d-75af-482d-b643-c5b04d3ffce5",
-    "version": "1.0"
+    "uuid": "d3ce3ac9-4180-4453-b288-e9d32a015da4",
```

Comment:
uuid doesn't change when we update the app. it remains same during life cycle of the app. Please revert back to original one